core/frontend/src/core-frontend.ts

```diff
@@ -209,6 +209,10 @@ export * from "./RealityDataSource";
  * @docs-group-description Tiles
  * Classes representing graphics as [hierarchical 3d tiles](https://github.com/CesiumGS/3d-tiles).
  */
+/**
+ * @docs-group-description RealityDataDisplayStyle
```

Comment:
Do we really need a separate doc group just for this?

Reply:
I had to add it to fix one of the iTwin.js Docs requirement. Is there a way to avoid it?